cbindgen.toml

```diff
@@ -7,10 +7,3 @@ header = """
 """
 include_version = true
 braces = "NextLine"
-
-[export]
-include = [
-  "ddg_up_classify_json",
-  "ddg_up_free_string",
-]
-
```

Comment:
I found that this wasn’t needed at least on macOS. Happy to revert this change and include the third function, but it still generates all three without this `include` statement.